setup.py

```diff
@@ -93,9 +93,7 @@ def run(self) -> None:
                 wheel = "macosx_10_13_x86_64.whl"
             if platform == "linux":
                 wheel = "manylinux1_x86_64.whl"
-            if platform == "win32":
-                wheel = "win32.whl"
-            if platform == "win32_x64":
+            if platform == "win":
```

Comment:
This would break 32 bit python installations on 64 bit windows, better to alias it to 64 bit and 32 bit to same wheel